bitnami/contour/values.yaml

```diff
@@ -178,6 +178,9 @@ contour:
   ## Create Contour CRDs
   installCRDs: true
 
+  ## Manage the creation and upgrade of Contour CRDs.
+  manageCRDs: true
+
```

Comment:
what is the difference between them?

Reply:
`manageCRDs` if set to true, will install, upgrade and delete CRDs along with the Helm chart. If the user wishes to manage their own CRDs, then they can set this to false.

It is used in `00-crds.yaml` -  
https://github.com/bitnami/charts/pull/3665/files#diff-32819b0a35e1ac9c445114f4fb1358eebaed346268f33652d4bb068507000803

Reply:
Yes, and what about `installCRDs`?

Reply:
`installCRDs` are an artefact from the Helm V2 days, is obsoleted and can be removed.